spotty/providers/aws/config/validation.py

```diff
@@ -30,6 +31,8 @@ def validate_instance_parameters(params: dict):
                                              ),
         Optional('managedPolicyArns', default=[]): [str],
         Optional('instanceProfileArn', default=None): str,
+        Optional('inboundIp', default="0.0.0.0/0"): Use(IPNetwork, error='"inboundIp" should be a valid IP '
```

Comment:
Just to keep all code in the same style, could you please use single quotes for strings everywhere?